src/org/rascalmpl/library/util/PathConfig.java

```diff
@@ -470,17 +470,6 @@ public static PathConfig fromSourceProjectRascalManifest(ISourceLocation manifes
             messages.append(Messages.info("Rascal version:" + RascalManifest.getRascalVersionNumber(), manifestRoot));
         }
 
-        if (!projectName.equals("rascal")) {
```

Comment:
there was a copy lower in the code, that would jarify the uri. so I removed this one. 

But it might be the case that this is the `INTERPRETER` mode, and the one below is the `COMPILER` mode.